screens/MatchingScreen.tsx

```diff
@@ -55,7 +56,9 @@ export default function MatchingScreen({
             </Pressable>
             <Pressable
               onPress={() => {
-                refresh();
+                sendFlirt(user.id).then(() => {
+                  refresh();
+                });
```

Comment:
Think this is better, rest looks good though
```suggestion
                sendFlirt(user.id).then(refresh);
```

Reply:
![image](https://user-images.githubusercontent.com/36244641/213120487-125aa72d-233f-4454-8de9-4469a3cd562b.png)